### ISSUE TITLE
Permissions are incorrect on profile template that is synced

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#7364]
**Describe the bug**
When editing a profile template in a connection profile in a cluster, the permissions are incorrect on the other cluster members but are correct on the cluster member where the edit is made

**To Reproduce**
1. In a cluster
2. Go in the default connection profile
3. Edit aup_text.html
4. Change the content to something else
5. Check the permissions on the other cluster members:
```
root@tests-v12-ga-2:~# ls -l /usr/local/pf/html/captive-portal/profile-templates/default/
total 4
-rw------- 1 pf pf 64 Nov  9 16:38 aup_text.html
```

**Expected behavior**
Permissions are consistent on all cluster members

**Comment by julsemaan** [comment]
Also impacts updating the HTTP/RADIUS certificates

**Comment by jrouzierinverse** [comment]
This has been fixed by c37b32b012a934282ecd91f10f1d09dd7e402a60 please retest

**Comment by julsemaan** [comment]
Works as expected now!
